src/main/scala/org/clulab/wm/eidos/context/TimeNormFinder.scala

```diff
@@ -30,149 +43,187 @@ object TimeNormFinder {
     }
     new TimeNormFinder(new TemporalNeuralParser, regexes)
   }
+
+  def getTimExs(odinMentions: Seq[Mention]): Seq[TimEx] = {
+    val reachableMentions = EidosMention.findReachableMentions(odinMentions)
+    val timExSeq: Seq[TimEx] = reachableMentions.flatMap { odinMention =>
+      odinMention.attachments.collect {
+        case attachment: Time => attachment.interval
+      }
+    }
+    val timExMap: IdentityHashMap[TimEx, Int] = timExSeq.foldLeft(new IdentityHashMap[TimEx, Int]()) { (identityHashMap, timEx) =>
+      identityHashMap.put(timEx, 0)
+      identityHashMap
+    }
+    val timExArray = timExMap
+        .keySet
+        .asScala
+        .toArray
+        .sortBy { timEx: TimEx => timEx.span }
+
+    timExArray
+  }
+
+  def getTimExs(odinMentions: Seq[Mention], sentences: Array[Sentence]): Array[Seq[TimEx]] = {
+    val timExs: Seq[TimEx] = getTimExs(odinMentions)
+    val alignedTimExs: Array[Seq[TimEx]] = sentences.map { sentence =>
+      val sentenceStart = sentence.startOffsets.head
+      val sentenceEnd = sentence.endOffsets.last
+
+      timExs.filter { timEx =>
+        sentenceStart <= timEx.span.start && timEx.span.end <= sentenceEnd
+      }
+    }
+    alignedTimExs
+  }
 }
 
 class TimeNormFinder(parser: TemporalNeuralParser, timeRegexes: Seq[Regex]) extends Finder {
-
   private val CONTEXT_WINDOW_SIZE = 20
   private val BATCH_SIZE = 40
 
   def parseBatch(text: String, spans: Array[(Int, Int)],
-                 textCreationTime: TimExInterval = UnknownInterval()): Array[Array[TimeExpression]] = {
+    textCreationTime: TimExInterval = UnknownInterval()): Array[Array[TimeExpression]] = {
```

Comment:
Was this de-denting intentional?

Reply:
Doubtful but not out of the question.  I would have double indented it as a continuation line, so maybe it was the IDE.  Will change it back just to be nice.